content/docs/goFSH/_index.md

```diff
@@ -3,8 +3,8 @@ title: "GoFSH"
 weight: 12
 ---
 
-This section contains documentation for GoFSH, a reference implementation FSH decompiler, which turns FHIR artifacts into FSH definitions.
+This section contains documentation for GoFSH, which turns FHIR artifacts into FSH definitions. Using GoFSH, you can turn an existing FHIR Implementation Guide into a FSH project automatically. If you are starting a project from scratch, see [Initializing a SUSHI Project](/docs/sushi/project/#initializing-a-sushi-project).
 
 {{% alert title="Warning" color="warning" %}}
-GoFSH is in the early stages of development, and is neither functionally complete nor bug-free. Use GoFSH with caution, reviewing all outputs before integrating them into FSH projects.
+GoFSH is in beta release, and does not yet convert value sets and instances to FSH.
```

Comment:
I feel like if we're too specific about what is and is not working this will be out of date all the time.  For example, next release will have support for most value sets and instances -- but there will still be some other things missing and/or non-ideal.  I preferred the old message that cautioned users to check their output.

In addition, since the releases are not tagged as `beta` but are just `0.x`, I think saying that GoFSH is in `pre-release` might be more accurate than `beta`.

Reply:
Pre-release is fine. I'll point people to the release notes.